recipe/meta.yaml

```diff
@@ -22,12 +22,12 @@ requirements:
     - python
     - poetry-core 1.5
   run:
-    - markdown-it-py >=2.2.0,<3.0.0
-    - pygments >=2.13.0,<3.0.0
+    - markdown-it-py >=2.2.0
+    - pygments >=2.13.0
     - python
-    - typing_extensions >=4.0.0,<5.0.0  # [py<39]
+    - typing_extensions >=4.0.0  # [py<39]
   run_constrained:
-    - ipywidgets >=7.5.1,<9
+    - ipywidgets >=7.5.1
```

Comment:
The upper bound is still relevant https://github.com/Textualize/rich/blob/7f580bdcf07a3b269a0e786b6a3aa9c804f393cf/pyproject.toml#L33

Reply:
same as https://github.com/AnacondaRecipes/rich-feedstock/pull/7#discussion_r1665396348